app/controllers/admin/settings/general_settings_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Admin
   module Settings
     class GeneralSettingsController < Admin::Settings::BaseController
+      after_action :bust_content_change_caches, only: %i[create]
```

Comment:
Given that this controller only has one action I'm very tempted to inline this right after the `Audit::Logger` call to make it a bit more obvious. But I'm happy to leave it here if others prefer that.